lua/nvim-web-devicons/icons-default.lua

```diff
@@ -1946,6 +2636,12 @@ local icons_by_file_extension = {
     cterm_color = "113",
     name = "Txt",
   },
+  ["ui"] = {
+    icon = "",
```

Comment:
This one appears to be QT; can we use `nf-linux-qt` please?

Reply:
There are also UI files for GTK, I don't know if they are cross-toolkit but seems to not be only Qt.

Reply:
It'll do.